api/src/api/comment/comment.routes.ts

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+
+import { CreateComment, GetComments, UpdateComment, DeleteComment } from "./comment.controllers";
+
+const router = express.Router();
+
+router.post("/createcomment", CreateComment);
+router.post("/getcomments", GetComments);
+router.post("/updatecomment", UpdateComment);
+router.post("/deletecomment", DeleteComment);
+
```

Comment:
We should be utilizing different methods:
- `GET comments/:id`: Get a comment
- `PUT comments/:id`: Update a comment 
- `DELETE comments/:id`: Update a comment 
- `POST comments`: Create a comment